09-features.md

```diff
@@ -42,8 +42,7 @@ The Context column decodes as follows:
 | 14/15 | `payment_secret`                  | Node supports `payment_secret` field                      | IN9      | `var_onion_optin`         | [Routing Onion Specification][bolt04]                                 |
 | 16/17 | `basic_mpp`                       | Node can receive basic multi-part payments                | IN9      | `payment_secret`          | [BOLT #4][bolt04-mpp]                                                 |
 | 18/19 | `option_support_large_channel`    | Can create large channels                                 | IN       |                           | [BOLT #2](02-peer-protocol.md#the-open_channel-message)               |
-| 20/21 | `option_anchor_outputs`           | Anchor outputs                                            | IN       | `option_static_remotekey` | [BOLT #3](03-transactions.md)                                         |
```

Comment:
What's the expected behavior if we peer sends bits `20/21` for a channel type open? Do we assume that it's actually `option_anchors_zero_fee_htlc_tx`, or fail as we've all removed the feature bits from our codebases so that shows up as an undefined combination?

Reply:
I think we should fail in that case, because that is likely an old node that supports the non-zero-fee variant so this won't work. A node that wants `option_anchors_zero_fee_htlc_tx` will just set the correct set of bits for it, right?

Reply:
If you still support it, you'd accept it?  But pretty soon, nobody will...